TruePath/FileEntryKind.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TruePath;
+/// <summary>
+/// Represents a kind of file system entry.
+/// </summary>
+public enum FileEntryKind
+{
+    /// <summary>An unknown kind of file system entry.</summary>
+    Unknown,
```

Comment:
I discussed the `Unknown` type with @ForNeVeR. We came to the conclusion that it is better not to use it: https://github.com/ForNeVeR/TruePath/issues/22